.github/workflows/ci.yml

```diff
@@ -53,5 +53,5 @@ jobs:
         with:
           stack-name: ${{ matrix.stack-name }}
           python-version: ${{ matrix.python-version }}
-          ref-zenml: ${{ inputs.ref-zenml || 'main' }}
+          ref-zenml: ${{ inputs.ref-zenml || 'feature/OSSK-342-rename-model-version-to-a-model' }}
```

Comment:
will this be changed later on?

Reply:
We change it every time needed, main runs `inputs.ref-zenml` and second option is for dev purposes